app/src/main/res/values-land/dimens.xml

```diff
@@ -65,6 +65,7 @@
   <!-- Content Item: Exploration View -->
   <dimen name="content_item_exploration_view_margin_start">48dp</dimen>
   <dimen name="content_item_exploration_view_margin_top">24dp</dimen>
+  <dimen name="content_item_exploration_view_margin_bottom">24dp</dimen>
```

Comment:
If this dimension is only needed for tests, could you instead create a new test-dimens.xml file for this to be part of? We should keep test-only artifacts separate from production ones.

(And ditto for the other values folders).

Reply:
Actually, please disregard this in favor of the suggestion below to hardcode the dimensions, instead.